tests/components/vicare/test_config_flow.py

```diff
@@ -203,29 +193,11 @@ async def test_form_dhcp(hass):
     assert len(mock_setup_entry.mock_calls) == 1
 
 
-async def test_import_already_configured(hass):
-    """Test that configuring same instance is rejectes."""
-    mock_entry = MockConfigEntry(
-        domain=DOMAIN,
-        unique_id="Configuration.yaml",
-        data=ENTRY_CONFIG,
-    )
-    mock_entry.add_to_hass(hass)
-
-    result = await hass.config_entries.flow.async_init(
-        DOMAIN,
-        context={"source": config_entries.SOURCE_IMPORT},
-        data=ENTRY_CONFIG,
-    )
-    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
-    assert result["reason"] == "already_configured"
-
-
 async def test_import_single_instance_allowed(hass):
     """Test that configuring more than one instance is rejected."""
     mock_entry = MockConfigEntry(
         domain=DOMAIN,
-        unique_id="Configuration.yaml",
+        unique_id=VICARE_NAME,
```

Comment:
The vicare name is not a unique id. We only set a unique id for the discovered entries.

Reply:
Without a unique id I cannot avoid that the user configures a 2nd configuration, right? Any suggestions how to prevent that?

Reply:
We already guard for more than one config entry:
https://github.com/home-assistant/core/blob/cfb47b9195ace9edf61dc1bcfd206a4005f8e4c4/homeassistant/components/vicare/config_flow.py#L43-L44